src/app/forms/components/weights/weights.component.ts

```diff
@@ -0,0 +1,46 @@
+import { Component, EventEmitter, Input, OnChanges, OnDestroy, OnInit, Output } from '@angular/core';
+import { FormArray } from '@angular/forms';
+import { DynamicFormService } from '@forms/services/dynamic-form.service';
+import { CustomFormGroup, FormNode } from '@forms/services/dynamic-form.types';
+import { Axle, TechRecordModel } from '@models/vehicle-tech-record.model';
+import { debounceTime, Subscription } from 'rxjs';
+
+@Component({
+  selector: 'app-weights',
+  templateUrl: './weights.component.html'
+})
+export class WeightsComponent implements OnInit, OnDestroy, OnChanges {
+  @Input() isEditable = false;
```

Comment:
Is it possible to alter the wording here slightly? `isEditable` refers to the configuration of the component, whether it is a view-only field or an editable field. Whereas we just want to track whether the user is currently editing the record. `isEditing` is present continuous and represents an ongoing action.
```suggestion
  @Input() isEditing = false;
```